tests/helpers/factories.py

```diff
@@ -24,3 +39,61 @@ def kspace_from_op(operator):
         operator.cpx_dtype
     )
     return kspace
+
+
+def to_interface(data, interface):
```

Comment:
SHouldnt this be avvailable to users ? I know it happens internally, but in case user wants to do something also?

Reply:
This mostly for helping the test setup. Inside MRI-nufft we have the `@with_numpy` decorator for all the backend working internally with numpy. For now, only cufinufft has some custom handling (for pytorch and cupy). 

Indeed such function are usefull, but IMO they are not central to the use of MRI-nufft (the user comes with its array, we take care of the rest). Maybe we could add them to more generic library like Modopt.